tests/linearGaussian_vector_field_test.py

```diff
@@ -91,26 +91,24 @@ def test_c2st_vector_field_on_linearGaussian(
     theta = prior.sample((num_simulations,))
     x = linear_gaussian(theta, likelihood_shift, likelihood_cov)
 
-    score_estimator = inference.append_simulations(theta, x).train(
-        training_batch_size=100,
-        max_num_epochs=50,
-    )
+    vf_estimator = inference.append_simulations(theta, x).train()
     # amortize the training when testing sample_with.
     for method in sample_with:
         posterior = inference.build_posterior(
-            score_estimator,
+            vf_estimator,
             sample_with=method,
             neural_ode_backend="zuko",
         )
         posterior.set_default_x(x_o)
         samples = posterior.sample((num_samples,))
 
         # Compute the c2st and assert it is near chance level of 0.5.
+        # Increased tolerance to 0.12 because FMPE can be around 0.61 for some runs.
```

Comment:
Did this improve with the better default hyperparameters? I find it a bit surprising that FMPE still struggles (relatively) on this task.

Reply:
Yeah this might be an artifact from before the changes. Will test.